alchemist/src/orchid/resources/wiki/usage/installation.md

```diff
@@ -0,0 +1,10 @@
+---
+
+title: Requirements and installation
+
+---
+
+Instructions about how to import the project can be found in the README.md file.
+
+***NOTE THAT***: Alchemist requires java 11+, so make sure to have a JDK version 11+ installed.
```

Comment:
Add a couple of things:

1. A link to adopt openjdk where to download the Java JDK
2. A link to the most recent travis build, saying that the JDKs and environment tested there are those for which executing Alchemist is supported
3. A link to Jabba, explaining that it's another possibility for quickly installing a JDK

Reply:
All three things added